src/index.ts

```diff
@@ -5,14 +5,57 @@ import dotenv from 'dotenv';
 
 dotenv.config()
 
+const DevIDs = ["170960451704717312"/*dislocated*/, "428934780764160010"/*tomkettle*/]
```

Comment:
Might be worth moving this to .env

Reply:
We can, yh, that might be tidier. Is there a nice way to hold arrays in .env or do you have to jank it?